packages/search/src/features/records/service.ts

```diff
@@ -693,6 +686,13 @@ export const aggregateRecords = ({
         bundle: { $concatArrays: ['$bundle', '$joinResult'] }
       }
     },
+    // Get DocumentReferences by Encounter ids
+    // (Encounter.id -> DocumentReference.subject.reference)
```

Comment:
This fix works because the Encounter $lookup is now done before the DocumentReference join. Why did it work earlier if 
>If the user unassigns and reassigns the request to themselves, they then see the correction supporting document.

this was done?

Reply:
That's a good question. Looking into the payloads should give us some clue. I'll look into it

Reply:
Found out the reason: https://github.com/opencrvs/opencrvs-core/blob/a5bf7105dacf1e4885a28b8e44db5e036a02f11d/packages/search/src/features/records/service.ts#L319-L326
Basically after downloading the record once, a history entry has the reference to the encounter which then prompts the Encounter to be included in the bundle

Reply:
Thanks for clarifying. Makes sense